iOS/Shimmer/component/component.yaml

```diff
@@ -1,4 +1,4 @@
-version: 1.0.0.2
+version: 1.0.2
```

Comment:
This file needs to have the is_shell set and to reference the nuget that is specified in the spec. 

This is an example -> https://github.com/newky2k/XamarinComponents/blob/dh_InAppSettingsKit/iOS/InAppSettingsKit/component/component.yaml#L22-L24